server/models/executor.js

```diff
@@ -1,14 +1,53 @@
+var assert = require('assert');
 var async = require('async');
+var debug = require('debug')('strong-mesh-models:service-manager');
+var instModelWatcher = require('../../lib/helper').instModelWatcher;
+var shouldWatch = require('../../lib/helper').shouldWatch;
 
 module.exports = function(Executor) {
 
   // Refer to server-service.js for description of instance vs currentInstance
   // vs data.
 
+  var name = 'executor';
```

Comment:
Why not just get this value from `Executor`? Im sure loopback has something like `Executor.modelName` (need to check docs).